src/routes/AppContent/index.tsx

```diff
@@ -88,7 +95,7 @@ const AppContent = () => {
     }, []);
 
     const getComponent = () => {
-        if ((isLoadingActiveAccount || !isFetched || !activeAccountData) && !isEndpointRoute) {
+        if ((isP2PSettingsLoading || isLoadingActiveAccount || !isFetched || !activeAccountData) && !isEndpointRoute) {
```

Comment:
for a brief moment, if i dont add this, the currency dropdown is blank, so this will make sure to wait until we get the data from p2p settings